source/autocomplete.txt

```diff
@@ -12,4 +12,9 @@ Autocomplete
    :depth: 1
    :class: singlecol
 
-Content on this page will be addressed in DOCSP-44964.
\ No newline at end of file
+Content on this page will be addressed in DOCSP-44964.
+
+Learn More
+----------
+
+- :ref:`intellij-run-java-queries`
```

Comment:
just a heads up you might need to rebase since this is showing up as a merge conflict!

Reply:
Noted -- thanks!